test/metabase/dashboard_subscription_test.clj

```diff
@@ -434,8 +434,9 @@
 
     :fixture
     (fn [{dashboard-id :dashboard-id} thunk]
-      (with-link-card-fixture-for-dashboard (t2/select-one Dashboard :id dashboard-id) [_]
-        (thunk)))
+      (mt/with-temporary-setting-values [site-name "Metabase Test"]
```

Comment:
this fixes another flake with subscription test.

[failing run](https://github.com/metabase/metabase/actions/runs/4382391846/jobs/7671394777#step:7:405)

in the last block, the test expects `Sent from Metabase Test`, but the actual result is `Sent from Metabase` (without "test"). bind here to make sure the settings is consistent.